lib/jxl/dec_frame.h

```diff
@@ -217,6 +192,18 @@ class FrameDecoder {
     if (use_task_id_) return task;
     return thread;
   }
+  bool CanDoRgbOutOrPixelCallback() const {
```

Comment:
Could also call it "CanDoLowMemoryPath()" or something like that :)

Can you please move/update the comment on MaybeSet...?

Reply:
Done.